scripts/switch.py

```diff
@@ -878,7 +879,7 @@ class SwitchHotswapHybrid(HotswapBase, CherryMXBase, Switch):
     ch_pin_dia = 1.5
 
     # TODO: Choc Hotswap pad size and locations do not match datasheet
-    hotswap_pad_size = vector(2.9, 2.6)
+    hotswap_pad_size = vector(2.75, 2.6)
```

Comment:
why is this? isn't this based on the existing hotswap chocs? because that hole size is for a reason, assembly, this is the value used by Keyboardio in production

Reply:
not sure what you are referring to here? hotswap pad size was slightly bigger (reverted after some fiddling with pad 2 placement). the hole sizes and placements should be the same if you overlayed the two footprints manually